articles/quickstart/native/react-native/_includes/_config.md

```diff
@@ -0,0 +1,21 @@
+## Configure Callback URLs
+
+Callback URLs are the URLs that Auth0 invokes after the authentication process. Auth0 routes your application back to this URL and appends additional parameters to it, including a token. Since callback URLs can be manipulated, you will need to add your application's URL to your client's **Allowed Callback URLs** for security. This will enable Auth0 to recognize these URLs as valid. If omitted, authentication will not be successful.
```

Comment:
At the end I'll add:
"You must add one callback URL per platform you're targeting". (as the android and ios URLs differ)